Ghiurutan V. Alexandru/Homework5/src/model/Word.java

```diff
@@ -0,0 +1,38 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Observable;
+
+public class Word extends Observable {
+
```

Comment:
Observer can and should be used when any update on the dictionary is made. That way, the user is notified of the changes.